install/operator/pkg/syndesis/action/upgrade.go

```diff
@@ -45,11 +46,7 @@ func (a *upgradeAction) CanExecute(syndesis *v1alpha1.Syndesis) bool {
 
 func (a *upgradeAction) Execute(ctx context.Context, syndesis *v1alpha1.Syndesis) error {
 	if a.operatorVersion == "" {
-		operatorVersion, err := template.GetSyndesisVersionFromOperatorTemplate(a.scheme)
-		if err != nil {
-			return err
-		}
-		a.operatorVersion = operatorVersion
+		a.operatorVersion = pkg.DefaultOperatorTag
```

Comment:
Is this related with this PR?

Reply:
We eliminated a bunch of variables from the conf.yaml, basically you don't get the operator version from the conf.yaml anymore.

Reply:
ahhh ... ok check